package.json

```diff
@@ -25,7 +25,8 @@
     "react": "15.4.2"
   },
   "dependencies": {
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
```

Comment:
Ideally you don't use both libraries. Both are relatively sizeable, and this code runs in the browser.